src/Driver/driver_configs.jl

```diff
@@ -404,6 +406,60 @@ Establishing Atmos GCM configuration for %s
     )
 end
 
+export EmptyParamSet
+export init_empty!, get_grid_spacing_les, get_grid_spacing_gcm
+struct EmptyParamSet <: AbstractEarthParameterSet end
```

Comment:
We'll need to remove this-- we're restricting these definitions to top-level files, outside of `src/`.